### ISSUE TITLE
ocopress unpublish gives error split' for nil:NilClass

### ISSUE THREAD

**Issue by Jeansen** [octopress/octopress#168]
When I do `octopress unpublish someposts` I am presented with a numbered list of all findings. I am asked `Which do you want to unpublish? (enter a number):` So, I enter a number, e.g. 1. I then get the error `octopress 3.0.11 | Error:  undefined method 'split' for nil:NilClass`

**Comment by jeremyz** [comment]
hello,
I tried to reproduce that with head which is 3.1.0, no can do.
can you please confirm it's fixed and close this issue, or give more information.

**Comment by jeremyz** [comment]
this might be related to #154

**Comment by Jeansen** [comment]
Nope, sorry. Does not work, though I changed to version 3.1.0. Here's my gemfile:
source "https://rubygems.org"
gem "jekyll", "~> 3.0"
gem "jekyll-sitemap"
gem "jekyll-feed"
gem 'octopress', :git=> 'git@github.com:octopress/octopress.git'
gem "jekyll-paginate"
gem "jekyll-gist"
gem 'github-pages', group: :jekyll_plugins

That's what I do: `bundle exec octopress unpublish post`
Found 8 posts matching: 'post'
  1) 2011-03-10-sample-post.md
  2) 2012-05-22-readability-post.md
  3) 2013-05-22-sample-post-images.md
  4) 2013-05-23-readability-feature-post.md
  5) 2013-06-25-video-post.md
  6) 2013-08-12-sample-link-post.md
  7) 2013-08-16-code-highlighting-post.md
  8) 2013-10-26-background-image.md
Which do you want to unpublish? (enter a number): 1

And that is what i get:
octopress 3.1.0 | Error:  undefined method `split' for nil:NilClass

**Comment by jeremyz** [comment]
OK got it,
your post is missing the mandatory date field, or it is not iso8601 formatted : 

date: 2016-07-20T13:57:39+02:00
